ansible/assessor_tools.yml

```diff
@@ -444,16 +444,6 @@
         assessment_tool_unarchive_extra_opts:
           - --strip-components=1
 
-    - name: Install resyncgg/ripgen
-      ansible.builtin.include_role:
-        name: assessment_tool
-      vars:
-        assessment_tool_archive_src: https://github.com/resyncgg/ripgen/tarball/main
-        assessment_tool_install_dir: /tools/ripgen
-        assessment_tool_rust: true
-        assessment_tool_unarchive_extra_opts:
-          - --strip-components=1
-
```

Comment:
Please update the PR description and title to reflect this change as well.

Reply:
I moved this commit to #179 and mentioned the change in that PR's description.

Reply:
That works for me, thanks.